integration/test/metrics_number_dimension/metrics_number_dimension_test.go

```diff
@@ -56,17 +54,26 @@ func TestNumberMetricDimension(t *testing.T) {
 			numberDimensionsInCW: 10,
 			metricName:           "mem_used_percent",
 		},
-		// @TODO add when https://github.com/aws/amazon-cloudwatch-agent/pull/361 is merged
-		// {resourcePath: "resources/30_dimension", findTarget: false, numberDimensionsInCW: 30, metricName: "mem_used_percent",},
-		// {resourcePath: "resources/35_dimension", findTarget: true, numberDimensionsInCW: 30, metricName: "mem_used_percent",},
+		{
+			resourcePath:         "resources/30_dimension",
+			findTarget:           false,
+			numberDimensionsInCW: 30,
+			metricName:           "mem_used_percent",
+		},
+		{
+			resourcePath:         "resources/35_dimension",
+			findTarget:           true,
+			numberDimensionsInCW: 30,
+			metricName:           "mem_used_percent",
+		},
 	}
 
 	for _, parameter := range parameters {
 		//before test run
 		log.Printf("resource file location %s find target %t input number dimension %d metric name %s",
 			parameter.resourcePath, parameter.findTarget, parameter.numberDimensionsInCW, parameter.metricName)
 
-		target := fmt.Sprintf(targetString, maxDimension, parameter.numberDimensionsInCW)
+		target := fmt.Sprintf(targetString, cwPlugin.MaxDimensions, parameter.numberDimensionsInCW)
 
 		t.Run(fmt.Sprintf("resource file location %s find target %t", parameter.resourcePath, parameter.findTarget), func(t *testing.T) {
```

Comment:
Instead of using `resource file location %s find target %t`, would be better to initialize a `name` into struct so it would clarify what each test would be instead of find the corresponding json.